README.md

```diff
@@ -161,11 +161,6 @@ void *spdm_test_client_init(void)
 
    The test integrator shall preprare a `test_config` and initialize it with required test cases.
    ```
-common_test_case_config_t m_spdm_test_group_version_configs[] = {

```

Comment:
Only `SPDM_RESPONDER_TEST_CASE_VERSION_INVALID_REQUEST` was removed. `SPDM_RESPONDER_TEST_CASE_VERSION_SUCCESS_10` is still there.

Reply:
thanks for pointing that out! fixed :)